src/main/java/com/softserveinc/ita/pageobjects/admin/MainMenu.java

```diff
@@ -10,4 +11,10 @@ public ProtocolPage openProtocolPage() {
 
         return new ProtocolPage();
     }
+
+    public String getLoggedUserName() {
+        return $$(xpath("//button[@class = 'mat-menu-trigger mat-button mat-button-base']"))
```

Comment:
you could try such locator: "//mat-nav-list[@role='navigation']/following-sibling::button[1]" instead of elements list

Reply:
I want to use one method for getting this username value both for admin and student pages.
There is no navigation menu on the student page, but there are text field ("d-tester") and buttons with username and language.
May I use such locator: "//span[text()='d-tester']/following-sibling::button[1]"? or is it better to avoid search by text?

Or maybe the whole idea of making this one method is wrong and it's better to create separate method for student page?